### PR TITLE
CommentUncommentService

### PR DESCRIPTION
**Pull request by vasily-kirichenko**
Fix https://github.com/Microsoft/visualfsharp/issues/1785, thanks @vladima

### CODE REVIEW

File: vsintegration/src/FSharp.Editor/CommentUncommentService.fs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+open Microsoft.CodeAnalysis.Shared.Extensions
+open Microsoft.CodeAnalysis.Text
+open Microsoft.CodeAnalysis.Text.Shared.Extensions
+open Microsoft.VisualStudio.Text
+open Microsoft.VisualStudio.Text.Operations
+open Microsoft.VisualStudio.Utilities
+open Microsoft.CodeAnalysis.Editor.Implementation.CommentSelection
+open Microsoft.CodeAnalysis.Host.Mef
+open Microsoft.CodeAnalysis
+open System.Composition
```
Comment (forki) [inline vsintegration/src/FSharp.Editor/CommentUncommentService.fs]: are all these namespaces needed?

Comment (vasily-kirichenko) [inline vsintegration/src/FSharp.Editor/CommentUncommentService.fs]: no idea. errors are not highlighted in vs 2015 on this project. Ionide highlight almost everything as errors...

**Comment by dsyme** [comment]
Awesome @vasily-kirichenko @vladima.  Will merge when green

**Comment by dsyme** [comment]
Thank you @vasily-kirichenko  and @vladima.  I can't wait until we merge all the VFPT features too :)

**Comment by vasily-kirichenko** [comment]
@dsyme I think coloring and "highlight current symbol in file" could be ported from VFPT quite easily. Re-implementing coloring could be a waste of time, there are many corner cases and we have a lot at tests for it.
